src/storage/heap_file.h

```diff
@@ -313,6 +313,10 @@ struct heap_operation_context
 
   /* Performance stat dump. */
   PERF_UTIME_TRACKER *time_track;
+
+  /* skip logging */
+  bool skip_logging;
```

Comment:
please move this after `is_bulk_op` and rename it as `use_page_logging` or `use_bulk_logging`.